cmd/gaia/testnets/README.md

```diff
@@ -151,6 +151,14 @@ MYPUBKEY=<your_newly_generated_public_key>
 
 The best way to get tokens is from the [Cosmos Testnet Faucet](https://faucetcosmos.network). If the faucet is not working for you, try asking [#cosmos-validators](https://riot.im/app/#/room/#cosmos-validators:matrix.org).
 
+After receiving tokens to your address, you can view your account's balance by typing: 
+
+```
+gaiacli account <your_newly_generated_address>
+```
+
+Note: If you query your account before any tokens have been sent to it, you will see a `panic` error. This error is expected! We are working on improving error messages.
```

Comment:
But this is fixed in the current version. (v0.18.0 has the fix)

Reply:
Ah ok! Good to know.